gerrit_changes_to_github.py

```diff
@@ -271,6 +178,87 @@ def is_repository_usable(args):
     return got_gerrit and got_target
 
 
+def get_changes_from_event_logs(args, gerrit_password, ghpa_token):
+    """Go through all events since the latest timestamp"""
+    try:
+        latest_timestamp = int(os.getenv("LAST_TIMESTAMP"))
+    except Exception as e:
+        print(f"Error reading last timestamp: {e}")
+
+    latest_ts_as_date = datetime.fromtimestamp(
+        latest_timestamp, tz=timezone.utc
+    ).strftime("%Y-%m-%d %H:%M:%S")
+    log.info(f"Using last timestamp (UTC): {latest_ts_as_date}")
+
+    # Set t2 cutoff to avoid getting a fraction of events for a particular second
+    t2 = int(datetime.now().timestamp()) - 1
+    t2_as_date = datetime.fromtimestamp(t2, tz=timezone.utc).strftime(
+        "%Y-%m-%d %H:%M:%S"
+    )
+
+    if (
+        events := requests.get(
+            f"https://review.spdk.io/a/plugins/"
```

Comment:
move this url to dispatch.yml as gerrit-api-url instead of deleting gerrit-api-url